packages/form-field/src/FormField/FormField.styles.ts

```diff
@@ -30,9 +30,12 @@ export const getFontSize = ({
   }
 };
 
-export const labelTextContainerStyle = css`
+export const textContainerStyle = css`
   display: flex;
   flex-direction: column;
+`;
+
+export const labelContainerStyle = css`
```

Comment:
could this instead be changed to `padding-bottom: ...` to avoid adding an extra div?

Reply:
Updated!